javascript-bank/bank.js

```diff
@@ -0,0 +1,43 @@
+/* exported Bank */
+function Bank() {
+  this.nextAccountNumber = 1;
+  this.accounts = [];
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+  if (Number.isInteger(balance) && balance > 0) {
+    var newAcc = new Account(this.nextAccountNumber, holder);
+    newAcc.deposit(balance);
+    this.accounts.push(newAcc);
+    this.nextAccountNumber++;
+    return this.nextAccountNumber - 1;
+  } else {
+    return null;
+  }
+};
+
+Bank.prototype.getAccount = function (number) {
+  for (var i = 0; i <= this.accounts.length; i++) {
+    if (i === this.accounts.length) {
+      return null;
+    } else if (this.accounts[i].number === number) {
+      return this.accounts[i];
+    }
+  }
+};
+
+Bank.prototype.getTotalAssets = function () {
+  var totalAssets = 0;
+  for (var i = 0; i <= this.accounts.length; i++) {
+    if (i === this.accounts.length) {
+      return 0;
```

Comment:
Instead of doing this, you can just put `return totalAssets` below the loop. If the array's length is 0, the loop would never start, which means you would return the initial value of `totalAssets`, which is 0.

Reply:
smart